### PR TITLE
Make the ECS a requireable module

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

File: jujugui/static/gui/src/app/init/environment-change-set.js [hunk 1]
```diff
@@ -0,0 +1,1637 @@
+/*
```
Comment (huwshimi) [inline jujugui/static/gui/src/app/init/environment-change-set.js]: Pretty much all indentation changes.

**Comment by huwshimi** [comment]
[Make the ECS a requireable module](https://trello.com/c/y1azBSo5/674-make-the-ecs-a-requireable-module)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8728/

**Comment by huwshimi** [comment]
Thanks @jaycee :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**